rescript/tests/test_ncbi.py

```diff
@@ -0,0 +1,36 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from qiime2 import Metadata
+from qiime2.plugin.testing import TestPluginBase
+from qiime2.plugins import rescript
+from pandas import DataFrame
+from q2_types.feature_data import DNAIterator
+
+
+class TestNCBI(TestPluginBase):
+    package = 'rescript.tests'
+
+    def test_get_ncbi_data(self):
+        df = DataFrame(index=['M59083.2', 'AJ234039.1'])
+        df.index.name = 'id'
+        md = Metadata(df)
+        get_ncbi_data = rescript.methods.get_ncbi_data
+        acc_seq, acc_tax = get_ncbi_data(accession_ids=md)
+        que_seq, que_tax = get_ncbi_data(query='M59083.2 OR AJ234039.1')
+
+        acc_seq = {s.metadata['id']: str(s) for s in acc_seq.view(DNAIterator)}
+        que_seq = {s.metadata['id']: str(s) for s in que_seq.view(DNAIterator)}
+        self.assertEqual(acc_seq, que_seq)
```

Comment:
could you add some expected results and then confirm that both acc and query methods retrieve those expected results? e.g., what if both queries are failing and output empty files... would these tests pass?

Reply:
as you wish